### PR TITLE
EPMRPP-96415 || Default number of items per page is displayed when us…

### PR DESCRIPTION
**Pull request by BlazarQSO**
…er returns to the page

## PR Checklist

* [x] Have you verified that the PR is pointing to the correct target branch? (`develop` for features/bugfixes, other if mentioned in the task)
* [x] Have you verified that your branch is consistent with the target branch and has no conflicts? (if not, make a rebase under the target branch)
* [x] Have you checked that everything works within the branch according to the task description and tested it locally?
* [x] Have you run the linter (`npm run lint`) prior to submission? Enable the git hook on commit in your IDE to run it and format the code automatically.
* [x] Have you run the tests locally and added/updated them if needed?
* [x] Have you checked that app can be built (`npm run build`)?
* [x] Have you made sure that all the necessary pipelines has been successfully completed?
* [x] If the task requires translations to be updated, have you done this by running the `manage:translations` script?

### CODE REVIEW

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/controllers/members/selectors.js [hunk 1]
```diff
@@ -25,4 +25,5 @@ export const loadingSelector = (state) => domainSelector(state).loading || false
 
 export const querySelector = createQueryParametersSelector({
```
[the remark this answers is not in the document]
Comment (BlazarQSO) [reply app/src/controllers/members/selectors.js:26]: This is an old pagination and can work via namespace:
image

But then there will be such parameters:
image

with an alternative namespace:
image

File: app/src/controllers/pagination/selectors.js [hunk 2]
```diff
@@ -14,11 +14,33 @@
...
+import { getAlternativePaginationAndSortParams } from './utils';
 
 export const totalElementsSelector = (paginationSelector) => (state) =>
   paginationSelector(state).totalElements;
 export const totalPagesSelector = (paginationSelector) => (state) =>
   paginationSelector(state).totalPages;
 
 export const defaultPaginationSelector = () => DEFAULT_PAGINATION;
+
+export const createParametersSelector = ({
```
[the remark this answers is not in the document]
Comment (BlazarQSO) [reply app/src/controllers/pagination/selectors.js]: Done.

File: app/src/controllers/pages/selectors.js [hunk 3]
```diff
@@ -105,6 +105,7 @@ export const prevPagePropertiesSelector = (
 
 export const createQueryParametersSelector = ({
   namespace: staticNamespace,
+  alternativeNamespace,
```
[the remark this answers is not in the document]
Comment (BlazarQSO) [reply app/src/controllers/pages/selectors.js:109]: When everyone switches to the same pagination format, then you can leave one option.

File: app/src/controllers/pagination/withPagination.jsx [hunk 4]
```diff
@@ -29,6 +29,7 @@ export const withPagination = ({
   paginationSelector = defaultPaginationSelector,
   namespace,
   namespaceSelector,
+  alternativeNamespace,
```
[the remark this answers is not in the document]
Comment (BlazarQSO) [reply app/src/controllers/pagination/withPagination.jsx:32]: When everyone switches to the same pagination format, then you can leave one option.

**Pull request closed without merging**